apps/web/screens/Search/Search.tsx

```diff
@@ -995,12 +995,19 @@ const EnglishResultsLink: FC<
     QuerySearchResultsArgs
   >(GET_SEARCH_RESULTS_TOTAL)
 
+  const excludedType = SearchableContentTypes.WebQna
+
+  const types = Object.values(SearchableContentTypes).filter(
+    (type) => !excludedType.includes(type),
```

Comment:
I feel it would make a bit more sense if we instead had a list of `excludedTypes` because as it stands right now we are checking if a type is a substring of the webqna string.